services/dashboard/tests/mocks/workflow.ts

```diff
@@ -0,0 +1,48 @@
+import { faker } from '@faker-js/faker'
+import { HttpResponse, http, type HttpHandler } from 'msw'
+
+const { json } = HttpResponse
+
+export const handlers: Array<HttpHandler> = [
+	http.get(`https://localhost:7777/workflow`, async () => {
+		console.info('🔶 mocking reactflow components:')
+
+        const place_1 = faker.location.city();
+        const place_2 = faker.location.city();
+
+		return json({
+			"nodes": [
```

Comment:
This space is causing ToC to me :p

Reply:
This is rather odd, in my IDE the code spacing is different, maybe because of extensions. I am going to reformat anyway.

Reply:
Yeah, we need to find a standard